.github/ISSUE_TEMPLATE/user_story.md

```diff
@@ -1,15 +1,27 @@
 ---
 name: User story
 about: Create a new user story for this project
-title: "[US] As [...], I'd like to [...], in order to [...]."
+title: "[US] [Name this US as it appears in the sprint goal]"
```

Comment:
I don't think [US] is useful, there's already the actual kind:user-story tag, which enables filtering just by clicking on it

Reply:
I know the tag allows us to filter the us among other issues, but I'd like to keep it to be able to identify them quickly

Reply:
For example in the board, where tags are not visible

Reply:
I agree with the idea of removing `[US]` for consistency. We don't use a textual tag for other issues.

Reply:
> I know the tag allows us to filter the us among other issues, but I'd like to keep it to be able to identify them quickly, for example in the board, where tags are not visible. Moreover [US] isn't really making the issue title much longer. I don't care removing it but personnaly i find it quite useful.

Managing simultaneously 2 ways of categorization is not a good practice. But I understand your point of view

Reply:
Alright, i'll change it then